Tone/component/analysis/DCMeter.ts

```diff
@@ -4,7 +4,9 @@ import { MeterBase, MeterBaseOptions } from "./MeterBase";
 export type DCMeterOptions = MeterBaseOptions;
 
 /**
- * DCMeter gets the raw value of the input signal at the current time.
+ * DCMeter gets the raw value of the input signal's waveform amplitude
+ * at the current time and displays it as a value between 0 and 1.
```

Comment:
I think the DCMeter is not limited to 0-1, it should be able to accept any number.

Reply:
When I use `.getValue()`, I see output values of -1 to 1 (which is an error and needs changed in this). The doc edit references that output, not the input. Am I understanding this incorrectly?

Reply:
I'm looking at this basic demo that I set up the other day: https://codepen.io/cjwit/pen/PozzrxP

Reply:
The audio output of PolySynth is in the audio range (-1 to 1), but if you connect something with a different value, you can see that the values are not limited to that range: 

```js
const meter = new Tone.DCMeter()
const signal = new Tone.Signal(4).connect(meter)
```

Reply:
This is obviously related to me learning the system. Since a Signal appears to represent a wide range of types of values, it isn't clear to me what your example represents. (As in, I'm not sure what the "4" is relating to.) Since the point isn't troubleshooting here, I removed that language and pushed the change to the PR.